jwst/exp_to_source/main.py

```diff
@@ -1,73 +1,71 @@
-import os
 import sys
 import argparse
+from pathlib import Path
 
 from stdatamodels.jwst.datamodels import MultiSlitModel
 
 from jwst.exp_to_source import exp_to_source
 
 
-class Main():
-    """Convert exposure-based slits data to source-based data
-    Docs from the source.
-
-    Parameters
-    ----------
-    args: str or [str,...]
-        The command-line arguments. This is passed to
-        `argparse.parse_args`. If None, `sys.argv`
-        is used.
+class Main:
+    """
+    Convert exposure-based slits data to source-based data.
 
-    Attributes
-    ----------
-    sources: {source: MultiExposureModel, ...}
-        A dict keyed on source name whose value is
-        the corresponding MultiExposureModel.
+    Docs from the source.
```

Comment:
What does this mean?

I think this should say something to the effect of, "Command-line interface to the exp_to_source step. For help, use `exp_to_source -h`. See also the `exp_to_source` readthedocs page; `https://link-to-page.html`"

Reply:
ok, thank you, that is clearer.